MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>Write at least a little comment at start of cell</p><p> </p><p>e.g.</p><p> </p><pre class="ql-syntax" spellcheck="false"># List columns which have at least one NA value
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>